card/db/migrate_core_cards/20170515101260_add_advanced_edit_cards.rb

```diff
@@ -0,0 +1,11 @@
+# -*- encoding : utf-8 -*-
+
+class AddAdvancedEditCards < Card::Migration::Core
+  def up
+    ensure_card "*activity toolbar button", codename: "activity_toolbar_button"
+    ensure_card "*rules toolbar button", codename: "rules_toolbar_button"
+    ensure_card "Anyone With Role",
+                codename: "anyone_with_role",
+                type_id: Card::RoleID
```

Comment:
I'm not sure if I understand what you don't understand. 
If you set a permission rule to that role then anyone with a role gets access ("Anyone Signed In" doesn't count).